src/common/opencl.c

```diff
@@ -1631,7 +1645,15 @@ int dt_opencl_build_program(const int dev, const int prog, const char *binname,
           char dup[PATH_MAX] = { 0 };
           g_strlcpy(dup, binname, sizeof(dup));
           char *bname = basename(dup);
+          #if defined(_WIN32)
```

Comment:
Spaces before `#if`.
You *really*  should follow https://github.com/darktable-org/darktable/blob/master/CONTRIBUTING.md, in particular https://github.com/darktable-org/darktable/blob/master/CONTRIBUTING.md#coding-style
Just set up clang-format hook and never worry about it again.